recipes/mbedtls/all/test_package/CMakeLists.txt

```
@@ -1,10 +1,10 @@
 cmake_minimum_required(VERSION 3.1)
-project(test_package)
+project(test_package C)
 
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup()
+conan_basic_setup(TARGETS)
 
 find_package(MbedTLS REQUIRED)
 
-add_executable(${PROJECT_NAME} test_package.cpp)
+add_executable(${PROJECT_NAME} test_package.c)
 target_link_libraries(${PROJECT_NAME} MbedTLS::MbedTLS)
```

```suggestion
target_link_libraries(${PROJECT_NAME} MbedTLS::mbedtls)
```

We can now use an official CMake imported target.

---

Can I ask why? I've seen a lot of packages start using the `_multi` generator on newer recipes.
Is it because it is closer on behavior to the CMakeDeps that will be the default on Conan 2.0?